packer/install-dependencies.sh

```diff
@@ -357,6 +358,7 @@ K8S_VERSIONS="
 1.12.8
 1.12.7
 1.12.7-azs
+1.11.10
 1.11.9
 1.11.9-azs
 1.11.8
```

Comment:
why not remove 1.11.8?

Reply:
Because I forgot to. :-) Should I leave in `1.11.8-azs` until there is a `1.11.10-azs`, or remove it as well?

Reply:
I'll leave that up to @jadarsie

Reply:
/hold

Reply:
let's keep 1.11.8-azs in there for now until we have a 1.11.10-azs

/hold cancel